recipes/qiime-meta/1.9.1/meta.yaml

```diff
@@ -0,0 +1,262 @@
+{% set name = "qiime" %}
+{% set version = "1.9.1" %}
+
+package:
+  name: qiime-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.python.org/packages/source/q/qiime/{{ name|lower }}-{{ version }}.tar.gz
+  sha256: c12dcdae1fe44762a49668276d7e93f487d87a0dc2ff8ca696ac0eff9d92a8a8
+
+build:
+  number: 4
+  noarch: generic
+  version: {{ version }}
+
+requirements:
+  run:
+    - {{ pin_subpackage('qiime', exact=True) }}
+    - blast-legacy ==2.2.22
+    - cd-hit ==4.6.6
+    - clearcut ==1.0.9
+    - clustalw ==2.1
+    - fastq-join ==1.3.1
+    - infernal ==1.0.2
+    - mafft ==7.305
+    - mothur ==1.36.1
+    - muscle ==3.8.1551
+    - r-optparse
+    - r-vegan # ==2.3_4
+    - r-base >=3,<4
+    - raxml ==8.2.9
+    - sortmerna ==2.1b
+    - vsearch ==2.3.4
+test:
+  # Test copied from openms output to make sure a test environment for this output is created.
+  imports:
+    - qiime
+    - qiime.util
+    - qiime.check_id_map
+    - qiime.sort
+  commands:
+    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
+    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
```

Comment:
You'll need to put conda in as a requirement for this to work in the container.

Reply:
Thanks for the tip. Actually I just copy-pasted this from the openms recipe. Is it needed at all?

Reply:
Nope